docs/hacking-guide/en/code-coverage-report.md

```diff
@@ -0,0 +1,37 @@
+# Code coverage report
+
+## Getting JaCoCo exec files
+
+Before you can generate code coverage report by JaCoCo tool,
+you need to get data about what lines of code were executed
```

Comment:
You're not telling how to generate this information ^^

how to generate the data about what lines where executed?